pkg/events/events.go

```diff
@@ -86,6 +86,15 @@ func (e *eventDefinitions) NamesToIDs() map[string]ID {
 	return namesToIds
 }
 
+func (e *eventDefinitions) GetID(eventName string) (ID, bool) {
```

Comment:
I use it later in the context filter to avoid a repeated map build.
I also think it's a useful helper to have, we could definitely use it to simplify code in other places, instead of doing the map build.